src/HtmlGenerator/Program.cs

```diff
@@ -22,6 +22,10 @@ private static void Main(string[] args)
             var projects = new List<string>();
             var properties = new Dictionary<string, string>();
             var emitAssemblyList = false;
+            var force = false;
+            var noBuiltInFederations = false;
+            var offlineFederations = new Dictionary<string, string>();
+            var federations = new HashSet<string>();

```

Comment:
I've noticed the federations field is never used anywhere. Where should it be read?